ykrt/src/internal.rs

```diff
@@ -0,0 +1,81 @@
+use std::ffi::{CString, c_void};
```

Comment:
If we choose to merge this, I think we should have a module-level doc comment explaining the design decision.